database.py

```diff
@@ -170,32 +170,34 @@ def names_with_infix(self, infix):
                 if infix in str(pokemon.get_name())]
 
     def __load_data(self):
-        # Load all the Pokemon data. This does not include the 'Extra' Pokemon.
-        with open(self.directory + "/./Data/pokemon.txt", 'r') as data_file:
-            for line in data_file:  # Load everything but the Pokemon from the 'Extra' folder.
-                identifier, _, name = line.strip().partition(' ')
-                identifier = '{:03}'.format(int(identifier))
-                region = self.__determine_region(identifier)
-                path = self.__determine_folder(identifier) + "/" + identifier + ".jpg"
-                pokemon = Pokemon(identifier, name.lower(), region, path)
+        """Load all the Pokemon data. This does not include the 'Extra' Pokemon."""
+        with open(self.directory + "/./Data/light-dark.txt", 'r') as data_file:
+            for i, line in enumerate(data_file):
+                name, _, dark_threshold = line.strip().partition(' ')
+                id = i + 1  # zero-based indexing --> one-based sequence numbers
+                identifier = '{:03}'.format(id)  # zero padded string
+                region = self.__determine_region(id)
+                path = self.__determine_folder(id) + "/" + identifier + ".jpg"
+                pokemon = Pokemon(identifier, name.lower(), region, path, dark_threshold)
                 self.__pokemon_list.append(pokemon)
                 self.__pokemon_dictionary[pokemon.get_name()] = pokemon
 
     def __load_extra(self):
-        # Load all the file names of the images in the Extra folder.
+        """Load all the file names of the images in the Extra folder."""
```

Comment:
Yes, yes, yes, I'm surprised how no one else noticed this and changed it to a doc string sooner.